web/src/settings_panel_menu.js

```diff
@@ -179,6 +185,16 @@ export class SettingsPanelMenu {
         const $panel = $(".settings-section" + sel);
         return $panel;
     }
+
+    get_valid_users_section_panel(key = "active-users-admin") {
+        browser_history.update(`#organization/user-list-admin/${key}`);
+        $(".user-settings-section").hide();
+        $(`[data-user-settings-section="${CSS.escape(key)}"]`).show();
+
+        const $current_tab = $("#admin-user-list .tab-switcher .ind-tab");
+        $current_tab.removeClass("selected");
+        $(`#admin-user-list .tab-switcher .ind-tab[data-tab-key="${key}"]`).addClass("selected");
```

Comment:
I think this can be removed once you do the changes I mentioned above in other comments.

Reply:
I have tried to do the changes in other comments but still this couldn't be removed.

Reply:
I guess we can directly call `toggler.goto` above by passing the `right_section` instead of having this function. `get_valid_users_section_panel` is mostly the code which is used in `goto`.

Reply:
Yeah definitely removing the `get_valid_users_section_panel` calls the toggler but displays all the three section. Even if i add a `toggler.goto` for first step , the main issue remains when we use `hashchange` to open the proper section. Because the toggler callback does not have any functionality to set the `selected` class for different tabs.

Reply:
You can just call `toggler.goto` and it will call `select_tab` and then `callback`. `select_tab` should handle `selected` class. I tried it and it works fine, but you should just test it once to make sure I did not miss any edge cases.